thorlcr/graph/thgraphslave.cpp

```diff
@@ -906,15 +906,16 @@ bool CSlaveGraph::recvActivityInitData(size32_t parentExtractSz, const byte *par
             actInitRtnData.clear();
             actInitRtnData.append(true);
             serializeThorException(e, actInitRtnData);
-            e->Release();
-            ret = false;
+            exception.setown(e);
```

Comment:
Why not throw here ?

Reply:
Because I still want to execute the block below that sends a response back to the master before the exception is fired.